.changeset/shiny-cows-share.md

```diff
@@ -0,0 +1,5 @@
+---
+'extract-react-types': minor
+---
+
+Add support for memos typed with generics
```

Comment:
and forwardRef?

Reply:
yeah forwardRef was already supported :)